### PR TITLE
smithy gradle plugin empty project

### PR DESCRIPTION
**Pull request by sbera87**
*Issue #, if available:*

*Description of changes:*

*Check all that applies:*
- [ ] Did a review by yourself.
- [ ] Added proper tests to cover this PR. (If tests are not applicable, explain.)
- [ ] Checked if this PR is a breaking (APIs have been changed) change.
- [ ] Checked if this PR will _not_ introduce cross-platform inconsistent behavior.
- [ ] Checked if this PR would require a ReadMe/Wiki update.

Check which platforms you have built SDK on to verify the correctness of this PR.
- [ ] Linux
- [ ] Windows
- [ ] Android
- [ ] MacOS
- [ ] IOS
- [ ] Other Platforms

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: tools/code-generation/smithy/codegen/gradle/wrapper/gradle-wrapper.jar
Comment (sbiscigl) [inline tools/code-generation/smithy/codegen/gradle/wrapper/gradle-wrapper.jar:1]: shouldnt commit jars to the repo, i would suggest adding it to your global git ignore.

Comment (sbiscigl) [reply tools/code-generation/smithy/codegen/gradle/wrapper/gradle-wrapper.jar:1]: > The scripts generated by this task are intended to be committed to your version control system. This task also generates a small gradle-wrapper.jar bootstrap JAR file and properties file which should also be committed to your VCS. The scripts delegates to this JAR.

im wrong needs to be there

Comment (sbera87) [reply tools/code-generation/smithy/codegen/gradle/wrapper/gradle-wrapper.jar:1]: seems like its a gradle wrapper requirement to have the jar

File: tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/SmithyCodegenPlugin.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.amazonaws.util.awsclientsmithygenerator.generators;
+
+import software.amazon.smithy.build.PluginContext;
```
Comment (sbiscigl) [inline tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/SmithyCodegenPlugin.java:3]: nit we dont need any of the imports yet, we shouldnt check them in with this revision

**Review by sbiscigl — APPROVED** [review]

File: tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/SmithyCodegenPlugin.java (hunk 1 above)
Comment (sbera87) [reply tools/code-generation/smithy/codegen/cpp-smoke-tests-codegen/src/main/java/com/amazonaws/util/awsclientsmithygenerator/generators/SmithyCodegenPlugin.java:3]: ack.removed

**Pull request merged**